woocommerce/woocommerce
# [Do Not Merge] Mothra explat workshop

This PR will be used for the Mothra meetup explat workshop.

I have created two experiments, but to try it out yourself feel free to modify the experiment names and create a new one yourself.
Experiments: 
- `explat_test_mothra_tip_of_the_day_focused` -> [Abacus link](https://experiments.a8c.com/experiments/20599-explat-test-mothra-tip-of-the-day-focused/overview)
- `explat_test_mothra_quote_of_the_day` -> [Abacus link](https://experiments.a8c.com/experiments/20600-explat-test-mothra-quote-of-the-day/overview)

### To Test
- Load this branch and install the [WooCommerce Test Helper](https://github.com/woocommerce/woocommerce-admin-test-helper/releases/tag/v0.7.2)
- Go to **WooCommerce > Home** and finish the onboarding
- Go to **Tools > WCA Test Helper > Experiments** and toggle the above mentioned experiments to see the difference.
- Reset the above experiments to `control` and now test it using the treatment bookmarklets from Abacus.
- Optional: append a unique id to each experiment name and create your own experiment in Abacus (make sure you always prepend it with `explat_test` as this is for testing purposes). Location of experiment names: [tip of the day](https://github.com/woocommerce/woocommerce/blob/demo/explat_workshop/plugins/woocommerce-admin/client/tip-of-the-day/tip-of-the-day.tsx#L14), [quote of the day](https://github.com/woocommerce/woocommerce/blob/demo/explat_workshop/plugins/woocommerce/src/Admin/API/QuoteOfTheDay.php#L86)
- Optional: Replace the front end use of `useExperiment` with either the `Experiment` component as mentioned [here](https://github.com/woocommerce/woocommerce/tree/demo/explat_workshop/packages/js/explat), or the `loadExperimentAssignment` as shown in the slides.
